autoload/leaderf/python/leaderf/bufExpl.py

```diff
@@ -39,9 +39,10 @@ def getContent(self, *args, **kwargs):
         bufnr_len = len(lfEval("bufnr('$')"))
         self._prefix_length = bufnr_len + 8
 
-        self._max_bufname_len = max(int(lfEval("strdisplaywidth('%s')"
-                                        % escQuote(getBasename(buffers[nr].name))))
-                                    for nr in mru.getMruBufnrs() if nr in buffers)
+        self._max_bufname_len = max((int(lfEval("strdisplaywidth('%s')"
+                                         % escQuote(getBasename(buffers[nr].name))))
+                                     for nr in mru.getMruBufnrs() if nr in buffers),
+                                    default=0)
```

Comment:
Thanks in advance. But max() does have "default" argument in python2 and python3 that version is less than 3.4.

Reply:
Perhaps the fix can be like this:
```python
        self._max_bufname_len = max([int(lfEval("strdisplaywidth('%s')"
                                        % escQuote(getBasename(buffers[nr].name))))
                                    for nr in mru.getMruBufnrs() if nr in buffers] or [0])
```